authui/src/authflowv2/base.css

```diff
@@ -32,4 +39,20 @@
     --c-grey-600: #303135;
     --c-grey-700: #131315;
   }
+
+  /* Border Radius */
+  :root {
+    --border-radius-full: 9999px;
+    --border-radius-sqaure: 0px;
+    --border-radius-rounded-square: 14px;
+  }
+
+  /* Typography */
+  :root {
+    --font-family-label-large: inherit;
```

Comment:
If we follow our convention of `primary-btn`, `label-large` should come first, making it `label-large-font-family`.

Reply:
@louischan-oursky What if naming it as `--t-label-large-font-family`, t for typography?